Source/MoreDotNet.Test/Extensions/Numeric/RomanNumeralExtensions/IsValidRomanNumeralTests.cs

```diff
@@ -0,0 +1,212 @@
+﻿namespace MoreDotNet.Tests.Extensions.Numeric.RomanNumeralExtensions
+{
+    using MoreDotNet.Extensions.Numeric;
+    using Xunit;
+
+    public class IsValidRomanNumeralTests
+    {
+        [Fact]
+        public void IsValidRomanNumeral_GivenI_ShouldReturnTrue()
```

Comment:
Please merge all the GvenI, GivenII, etc. tests into one, using XUnit Data Driven Tests using the [Theroy] and [MemberData] attributes

See: http://stackoverflow.com/questions/22093843/pass-complex-parameters-to-theory 
Note: PropertyData was replaced with MemberData in the newer XUnit versions, but the example is still applicable 

For more detailed info, see: http://ikeptwalking.com/writing-data-driven-tests-using-xunit/